src/test/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsMappingTest.java

```diff
@@ -93,6 +96,10 @@ void updateIds() {
         assertNotNull(wrapper.getCaseData().getOtherParties().get(0).getValue().getRep().getId());
 
         assertNotNull(wrapper.getCaseData().getOtherParties().get(1).getValue().getId());
+
+        assertNotNull(wrapper.getCaseData().getJointParty().getId());
+        assertNotNull(wrapper.getCaseData().getJointParty().getAppointee().getId());
```

Comment:
Hamcrest is better because it's easier to read and you get better error messages - assertThat(???).isNotNull();